src/app/hooks/use-top-filter-bar-click-outside-logic.tsx

```diff
@@ -7,7 +7,7 @@ import { useContentPaperRef } from 'app/layouts/PageLayout/context';
 
 import { useAssetsViewState } from './use-assets-view-state';
 
-export const useManageAssetsClickOutsideLogic = () => {
+export const useTopFilterBarClickOutsideLogic = () => {
```

Comment:
deleted this hook [here](https://github.com/madfish-solutions/templewallet-extension/pull/1276/files#diff-31adb6315fde75530f3d964e2a09c7b9c3e280daa5a52c263fc6d0fe1002c2faL2), check if you still need it